crm/modules/member/command.inc.php

```diff
@@ -308,7 +308,8 @@ function command_member_membership_delete () {
     }
     // Delete membership
     member_membership_delete($esc_post['sid']);
-    return crm_url("contact&cid=$_POST[cid]&tab=plan");
+    message_register("Deleted membership");
```

Comment:
This should be in crm/modules/member/data.inc.php in the "member_membership_delete" function, to be consistent with the "member_plan_delete" function